### PR TITLE
MAGN-9387 Remove preview bubble from Watch Node

### PR DESCRIPTION
**Pull request by pbidenko**
### Purpose

Preview bubble got removed from Watch node.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.

### Reviewers
@ramramps

### CODE REVIEW

**Comment by ramramps** [comment]
@pbidenko - Is this applicable for Watch and Watch3D?

**Comment by ramramps** [comment]
@pbidenko - There should be a test for this.

**Comment by pbidenko** [comment]
@ramramps yes, this PR is related to Watch node. A test was added. Please take another look.

File: test/DynamoCoreWpfTests/PreviewBubbleTests.cs [hunk 1]
```diff
@@ -135,5 +135,26 @@ public void PreviewBubble_CloseWhenFocusInCodeBlock()
...
+
+        [Test]
+        public void Watch_PreviewAllowanceDisabled()
+        {
+            OpenAndRun(@"core\WatchPreviewBubble.dyn");
+
+            var nodeView = NodeViewWithGuid("456e57f3-d06f-4a53-9771-27188ee9cb40");
+
+            // Raise mouse enter event.
+            View.Dispatcher.Invoke(() =>
```
Comment (ramramps) [inline test/DynamoCoreWpfTests/PreviewBubbleTests.cs]: I like it.

**Comment by ramramps** [comment]
LGTM

**Pull request merged**